BTTWriterCatalog/Models/DataModel/SupplimentalResourcesModel.cs

```diff
@@ -1,13 +1,12 @@
-﻿using Newtonsoft.Json;
-using System;
+﻿using System;
 using System.Collections.Generic;
-using System.Text;
+using System.Text.Json.Serialization;
 
 namespace BTTWriterCatalog.Models.DataModel
 {
     public class SupplimentalResourcesModel

```

Comment:
Typo: "supplemental" ("lem", not "lim")

Reply:
I'll fix that in the next PR, don't' want to deal with more rebasing at this point